astrocut/cube_cut.py

```diff
@@ -447,45 +448,48 @@ def _get_cutout(self, transposed_cube, threads: Union[int, Literal["auto"]] = 1,
             cutout = transposed_cube[xmin:xmax, ymin:ymax, :, :]
 
         img_cutout = cutout[:, :, :, 0].transpose((2, 0, 1))
-        uncert_cutout = cutout[:, :, :, 1].transpose((2, 0, 1))
+        if self.product == "SPOC":
+            uncert_cutout = cutout[:, :, :, 1].transpose((2, 0, 1))
     
         # Making the aperture array
         aperture = np.ones((xmax-xmin, ymax-ymin), dtype=np.int32)
 
         # Adding padding to the cutouts so that it's the expected size
         if padding.any():  # only do if we need to pad
             img_cutout = np.pad(img_cutout, padding, 'constant', constant_values=np.nan)
-            uncert_cutout = np.pad(uncert_cutout, padding, 'constant', constant_values=np.nan)
+            if self.product == "SPOC":
+                uncert_cutout = np.pad(uncert_cutout, padding, 'constant', constant_values=np.nan)
             aperture = np.pad(aperture, padding[1:], 'constant', constant_values=0)
 
         if verbose:
             print("Image cutout cube shape: {}".format(img_cutout.shape))
-            print("Uncertainty cutout cube shape: {}".format(uncert_cutout.shape))
+            if self.product == "SPOC":
+                print("Uncertainty cutout cube shape: {}".format(uncert_cutout.shape))
     
-        return img_cutout, uncert_cutout, aperture
+        if self.product == "SPOC":
```

Comment:
returning different amounts of things instead of always the same amount of things makes logic of dealing with the return values much harder.

the logic of dealing with spoc vs. tica is spreading throughout the codebase.

what do you think about initializing `uncert_cutout` to `None` and return all three variables no matter what?

we also need to update the docstring about the return value with whatever this function ends up doing.